azurelinuxagent/ga/update.py

```diff
@@ -806,21 +803,6 @@ def _ensure_no_orphans(self, orphan_wait_interval=ORPHAN_WAIT_INTERVAL):
                     ustr(e))
         return
 
-    def _ensure_partition_assigned(self):
```

Comment:
It seems like at some point there was some functionality to split agent updates across "partitions", and VMs were assigned to a particular partition. Not used anymore, so deleting this code.